executor/src/witgen/machines/double_sorted_witness_machine.rs

```diff
@@ -258,9 +258,7 @@ impl<T: FieldElement> DoubleSortedWitnesses<T> {
                 addr,
                 value
             );
-            let ass = (left[2].clone() - (*value).into())
-                .solve()
-                .map_err(|()| EvalError::ConstraintUnsatisfiable(String::new()))?;
+            let ass = (left[2].clone() - (*value).into()).solve()?;
```

Comment:
I remember this being done this way because the cause of the error is set by the caller. Has this changed so that it can be simplified like this? I like this more but just making sure.